recipes/pandas-vet/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pandas-vet" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57ea4906c176d61ea9c186607be7a53f4f1561b53f19f5b76ce2a5050f0bad41
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - attrs
+    - flake8 >3.0.0
+    - python >=3.6
+    - setuptools
+
+test:
+  imports:
+    - pandas_vet
+  requires:
```

Comment:
As this is only doing an import test I suspect these dependencies aren't needed? If they are required they should probably be runtime dependencies.